test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -470,17 +470,31 @@ class test_uct_cm_sockaddr : public uct_test {
                        const uct_cm_listener_conn_request_args_t
                        *conn_req_args) {
         test_uct_cm_sockaddr *self = reinterpret_cast<test_uct_cm_sockaddr *>(arg);
+        ucs_sock_addr_t m_connect_addr_sock_addr =
+                        self->m_connect_addr.to_ucs_sock_addr();
         uct_conn_request_h conn_request;
         const uct_cm_remote_data_t *remote_data;
+        uint16_t client_port;
         ucs_status_t status;
 
         EXPECT_TRUE(ucs_test_all_flags(conn_req_args->field_mask,
                                        (UCT_CM_LISTENER_CONN_REQUEST_ARGS_FIELD_CONN_REQUEST |
-                                        UCT_CM_LISTENER_CONN_REQUEST_ARGS_FIELD_REMOTE_DATA)));
+                                        UCT_CM_LISTENER_CONN_REQUEST_ARGS_FIELD_REMOTE_DATA  |
+                                        UCT_CM_LISTENER_CONN_REQUEST_ARGS_FIELD_CLIENT_ADDR)));
 
         conn_request = conn_req_args->conn_request;
         remote_data  = conn_req_args->remote_data;
 
+        /* check the address of the remote client */
+        EXPECT_EQ(memcmp(ucs_sockaddr_get_inet_addr(m_connect_addr_sock_addr.addr),
```

Comment:
can use `ucs::sock_addr_storage` class with `operator==` to improve readability:
```
EXPECT_EQ(self->m_connect_addr, ucs::sock_addr_storage(conn_req_args->client_address));
```

Reply:
> can use `ucs::sock_addr_storage` class with `operator==` to improve readability:
> 
> ```
> EXPECT_EQ(self->m_connect_addr, ucs::sock_addr_storage(conn_req_args->client_address));
> ```

the ports are different.

Reply:
can create a copy and reset ports to 0

Reply:
why not check only IPs like it is now?

Reply:
actually it was just suggestion to improve readability, but if check only IPs then need to change `EXPECT_EQ(0, memcpy(...))` because expected value should go first